docs/user_guide/installation.rst

```diff
@@ -17,7 +17,7 @@ To install the package in any generic Python v3.7-v3.9 environment, simply type
     If the standard Python Package Index (PyPI, where `pip` installs from) is not working for you, it
     should also be possible to install from conda-forge via
 
-    .. :code-block:: bash
+::
 
         conda install nwbinspector
```

Comment:
is this true? This is not working for me, and I don't recall you or I adding this to conda forge. Even if one of us had, the command for this would be `conda install --channel=conda-forge nwbinspector`

Reply:
Ahh, I thought I heard that conda automatically copied things from PyPI over to their service, guess I should have checked that.

Removed now.